src/helm/proxy/models.py

```diff
@@ -293,6 +301,15 @@ def engine(self) -> str:
         description="GPT-J (6B parameters) autoregressive language model trained on The Pile.",
         tags=[TEXT_MODEL_TAG, FULL_FUNCTIONALITY_TEXT_MODEL_TAG, GPTJ_TOKENIZER_TAG],
     ),
+    Model(
+        group="huggingface",
+        creator_organization="BigCode",
+        name="huggingface/santacoder",
+        display_name="SantaCoder (1.1B)",
+        description="SantaCoder (1.1B parameters) model trained on the Python, Java, and "
```

Comment:
In a follow-up PR, I will update it so we only have to specify these fields in a single place.